to.etc.domui/src/main/java/to/etc/domui/component/layout/ErrorMessageDiv.java

```diff
@@ -126,6 +128,14 @@ protected NodeContainer createErrorLine(UIMessage m) {
 		return d;
 	}
 
+	/**
+	 * Adds css selector for additional styling to div container.
+	 */
+	private void addAdditionalStyling(@Nonnull UIMessage m) {
+		if(getChildCount() >= 0 && getVisibility() == VISIBLE)
```

Comment:
getChildCount() >= is always true. The visibility should not matter as far as the extra class is concerned.